examples/web_testing/package.json

```diff
@@ -8,6 +8,7 @@
     "karma-chrome-launcher": "2.2.0",
     "karma-firefox-launcher": "1.1.0",
     "karma-jasmine": "2.0.1",
+    "karma-junit-reporter": "2.0.1",
```

Comment:
I think this is the one example where I'd expect to see the new feature working

Reply:
See previous comment.